tools/js-sdk-release-tools/src/test/apiVersion/apiVersionExtractor.test.ts

```diff
@@ -41,6 +45,13 @@ describe('Modular client api-version Extractor', () => {
         const version = await getApiVersionType(root);
         expect(version).toBe(ApiVersionType.Stable);
     });
+    test('update package version for userAgentInfo',async ()=>{
```

Comment:
you should create a new file `changeClientFile.test.ts`
and test all client types